src/common/logger.ts

```diff
@@ -22,6 +22,18 @@ interface IClientLogMsg {
 
 const MAX_LOG_QUEUE_LENGTH = 100;
 
+// Electron wants this to be called initially before calling
+// app.getPath('logs')
+app.setAppLogsPath();
+
+// Default log paths
+const MAC_LOGS_PATH = '/Library/Logs/Symphony/';
+const LINUX_LOGS_PATH = '/.config/Symphony/';
+const WINDOWS_LOGS_PATH = '\\AppData\\Roaming\\Symphony\\logs';
```

Comment:
please change this to `AppData\\Local` as we are making changes accordingly.

Reply:
The default location for logs in `6.x`+  has defaulted to `userData` path

Reply:
Yes, understood. But, we need to override that now to point to `local` rather than `roaming`.
Electron by default picks roaming for user data path rather than local which we need to override now.